common/config/azure-pipelines/jobs/fast-ci.yaml

```diff
@@ -35,8 +35,37 @@ pr:
       - "**/CHANGELOG.json"
       - "**/CHANGELOG.md"
 
+resources:
+  repositories:
+    - repository: build-pipeline-scripts
+      type: git
+      name: iModelTechnologies/imodeljs-build-pipeline-scripts
+
 jobs:
+  - job: VeracodeScan
+    condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+    displayName: Veracode SAST Scan
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+      - template: templates/appsec/veracode/package-veracode.yaml@build-pipeline-scripts
+        parameters:
+          foldersToInclude: "core,editor,domains,example-code,extensions,presentation,test-apps,tools,ui,utils" #note I skipped common, docs, full-stack-tests
+          foldersToExclude: "test"
```

Comment:
I saw in the CI run that files in subdirs `public` folders are being copied - should we exclude those as well?

Reply:
It doesn't seem there are any code files in those folders so that makes sense, I pushed that change.